docs/book/src/clusterctl/provider-contract.md

```diff
@@ -81,28 +81,23 @@ releaseSeries:
 
 <aside class="note">
 
-<h1> Embedded metadata </h1>
-
-The `clusterctl` command can ship with embedded metadata for pre-defined providers.
-If, as a provider implementer, you are interested to this feature, please send a PR to the [Cluster API repository](https://sigs.k8s.io/cluster-api).
-
-</aside>
-
-<aside class="note">
-
 <h1> Note on user experience </h1>
 
 If provider implementers only update the clusterctl's built-in metadata and don't
 provide a `metadata.yaml` in a new release, users are forced to update `clusterctl`
 to the latest released version in order to properly install the provider.
 
 As a related example, see the details in [issue 3418].
+
+To address above explained issue, the embedded metadata within clusterctl has been removed to prevent the reliance on using the latest version of clusterctl in order to pull newer provider releases.
```

Comment:
```suggestion
To address the above issue, the embedded metadata within clusterctl has been removed (as of v1alpha4) to prevent the reliance on using the latest version of clusterctl in order to pull newer provider releases.
```